### ISSUE TITLE
Lotus::Action::Parameters converting to hash

### ISSUE THREAD

**Issue by bradrobertson** [lotus/controller#113]
right now if I have something like:

```ruby
class LotusParams < Lotus::Action::Params
  param :contact do
    param :email, presence: true, format: /.+@.+/
  end
end
```

And I call
```ruby
p = LotusParams.new(contact: {email: 'xxx'})
p.to_h
```
I get
```ruby
{"contact"=>#<#<Class:0x007fe5e222f798>:0x007fe5e219d7d0 @env={"email"=>"xxx"}, @raw=#<Lotus::Utils::Attributes:0x007fe5e219d690 @attributes={"email"=>"xxx"}>, @attributes=#<Lotus::Utils::Attributes:0x007fe5e219d4b0 @attributes={"email"=>"xxx"}>>}
```

I'd like to actually convert this to a raw hash of ruby primitives as specified in these params data types. I looked at the `raw` method, but that doesn't return the whitelist or anything, it's just the ones passed in.

Is there an easy way to convert a params object back to a validated, whitelisted raw hash?

**Comment by bradrobertson** [comment]
If you're looking for context here on the usage, I want to start using Lotus in little bits as much as possible, but I'm not ready for the the framework as a whole.

My first thought is to write a little rails gem using these param validation classes that would allow some to specify the required params for a rails action and get a `validated_params` in their action method assuming it passed the conditions. These `validated_params` should ultimately be the final representation of the request params as a raw hash, rather than a `Lotus::Action::Params` object

**Comment by g3d** [comment]
Got a similar error with passing params into initializer for worker/service.

**Comment by mjbellantoni** [comment]
Pull #115 has a proposed fix for this.

**Comment by jodosha** [comment]
Fixed by #115
